src/vehicle.cpp

```diff
@@ -6545,7 +6545,12 @@ void vehicle::use_washing_machine( int p ) {
             n.set_age( 0 );
         }
 
-        drain( "water", 24 );
+        if ( fuel_left( "water" ) >= 24 ) {
```

Comment:
`if` should be glued to the opening bracket.

Reply:
Will fix this today.